data/names.txt

```diff
@@ -265,13 +308,24 @@ phrase "nouns of indeterminate quantity"
 		"Mirth"
 		"Olympus"
 		"Light"
+		"Charity"
+		"Mercy"
+		"Hope"
+		"Virtue"
+		"Fortitude"
+		"Enlightenment"
+		"Might"
+		"Destiny"
+		"Grass"
 
 phrase "the-nouns"
 	word
 		"the "
 	phrase
 		"singular nouns"
+		"plural nouns"
 		"space nouns"
+		"cardinal directions"
```

Comment:
Give it a whirl and see for yourself. I tested this extensively and didn't notice an excessive number of ships with cardinal directions in the name.